app/components/edit/review_step_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render Edit::StepComponent.new(edit_label: 'Undo your confirmation', data:, done_data:, edit_data:, show:, **args) do |component| %>
```

Comment:
The review step component is extracted from the step component. Steps 1 and 3 are review steps (the user doesn't change any inputs).